package.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "assertion-tools",
-  "version": "2.2.1",
+  "version": "8.0.0-gama.0",
```

Comment:
i dont really care personally but its gamma, idk how we have it on other projects